node/network/src/peer_manager.rs

```diff
@@ -138,7 +138,23 @@ impl PeerManager {
         Self { all_peer_states }
     }
 
-    /// Get channel for the given `account_id`, if the corrresponding peer is `Ready`.
+    /// This should be called if peer has done something wrong.
+    pub fn suspect_malicious(&self, peer_id: &PeerId) {
+        // TODO: add counter + banning (disconnect and don't connect again) flag here for given peers.
```

Comment:
I think in case of the network it is ok to have TODOs without issue ids, because most of the progress is tracked through the nomicon. If we create issues for each item we would immediately end up with a dozen or two of new issues.